src/Monticello/MCOrganizationDefinition.class.st

```diff
@@ -46,12 +64,14 @@ MCOrganizationDefinition >> basicCommonPrefix [
 
 { #category : #accessing }
 MCOrganizationDefinition >> categories [
-	^ categories
+
+	^ categories ifNil: [ { self packageName } , (self tagNames collect: [ :tagName | self packageName , '-' , tagName ]) ]
 ]
 
-{ #category : #accessing }
+{ #category : #deprecated }
 MCOrganizationDefinition >> categories: aCollection [
-	"ensure the categories are sorted alphabetically, so the merge don't take it as a conflict"
+	"Do not use. This methods should be replaced by #packageName and #tagNames"
```

Comment:
Deprecated but still used, even in this PR? In `MCPackage >> snapshot` and `MCStReader >> systemOrganizationFromRecords:`.
I assume `MCOrganizationDefinition class >> categories:` should also be deprecated.

Reply:
Yes, it will take some time to totally remove the usage. It’s the reason I just changed the protocol and I didn’t call the method #deprecated: , I do not want to raise warnings. I just want people not to introduce new senders to this method